docs/Server.md

```diff
@@ -418,6 +418,33 @@ Set a default
 Note that this is needed to offer the graceful "close" experience when
 using http2. Node core defaults this to `0`.
 
+<a name="framework-errors"></a>
+### `frameworkErrors`
+
++ Default: `null`
+
+Fastify provides default error handlers for the most common use cases.
+Using this option it is possible to override one or more of those handlers with custom code.
+
+*Note: Only `FST_ERR_BAD_URL` is implemented at the moment.*
+
+```js
+const fastify = require('fastify')({
+  frameworkErrors: function (error, req, res) {
```

Comment:
the user is not aware that:
- he must `res.end()` otherwise, the request will be pending 
- req and res are not the usual fastify request and reply object: this will depend on the error in the future I think

Reply:
Maybe we could provide them? 🤔

Reply:
We can still have our request/reply objects, in the same way we are doing for the 404 errors.